templates/pages/setup/general/api_apiclients_section.html.twig

```diff
@@ -33,7 +33,7 @@
 
 {% import 'components/form/fields_macros.html.twig' as fields %}
 
-{{ fields.largeTitle('APIClient'|itemtype_name(get_plural_number()), 'APIClient'|itemtype_icon) }}
+{{ fields.largeTitle(__('API clients (Legacy API)'), 'APIClient'|itemtype_icon) }}
```

Comment:
Not sure what our policy is on that but I dislike the name "Legacy API".

It sound like an old API that is deprecated and replaced by a new one that is not the case as far as I understand.
Low level API / High level API would be more precise but it sound too technical so unsure about it.

Reply:
"It sound like an old API that is deprecated and replaced by a new one" => maybe I did not understand well, but that was what I had in mind.

Reply:
I don't think there is a plan yet to completely remove the original API, but there is certainly a very hard push to get people using the new one instead. We even warn users now when they have the original API enabled. So, certainly deprecated starting in GLPI 11.

If anything the new API could stand to have a better name.

Reply:
See #17906

Reply:
> It sound like an old API that is deprecated and replaced by a new one

It's the plan. We don't remove the previous, but discourage usage for new integration.